scripts/python/qa-scripts/options.json.template

```diff
@@ -0,0 +1,60 @@
+{
+  "accountEmail": "",
+  "gSheetAuthFile": "gsheet-accounts/service_account.json",
+  "allowedBlankPerAssessor": 1,
+  "minimumVCA": 1,
+  "criteriaGroups": [
+    {
+      "1": "Impact / Alignment",
+      "2": "Feasibility",
+      "6": "Auditability"
+    },
+    {
+      "5": "Impact / Alignment",
+      "3": "Feasibility",
+      "4": "Auditability"
+    }
+  ],
+  "assessmentsSheet": "Assessments",
+  "proposalKeyCol": "Idea Title",
```

Comment:
Could you elaborate on why is this used?

Reply:
- `allowedBlankPerAssessor` is used to check the blanks rate of CAs. In previous funds we had a max blanks ratio allowed.
- `minimumVCA` is the minimum number of vCA needed to be able to calculate consensus. In previous funds it was > 1.
- `criteriaGroups` is the map for the questions from the original IdeaScale export and the criteria columns we use.

the other values are the names of the columns used in the various sheets to extract the info needed and create the columns for each next step. 
in last part of `options.json` there are the google sheet links used in the various steps and the title of the generated docs.

Reply:
I mean what's the purpose of the `proposalKeyCol` column? I saw that used in the scripts processing but I would expect it to be there for UX reasons only

Reply:
Yes, it is used only to present the name of the proposal for each assessment in the files generated.

Reply:
It's actually used as a group by key in https://github.com/input-output-hk/catalyst-toolbox/blob/85f7377637645b05ba9c9115af82329c354d295d/scripts/python/qa-scripts/createVCAAggregate.py#L172, that's what got me interested in this